### PR TITLE
Add a comment explaining some of the sequence parsing code.

### PR DESCRIPTION
**Pull request by phrohdoh**
I'm fine with closing this (or better, rewording!) if no one feels this is necessary, but it has confused me multiple times.

### CODE REVIEW

File: OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs [hunk 1]
```diff
@@ -37,6 +37,20 @@ public virtual ISpriteSequence CreateSequence(ModData modData, TileSet tileSet,
 				nodes.Remove("Defaults");
 				nodes = nodes.ToDictionary(kv => kv.Key, kv => MiniYaml.MergeStrict(kv.Value, defaults));
 
+				// Merge default animation image value. An example follows.
+				// Note: This example isn't entirely true as we remove the 'Defaults' section above.
```
Comment (pchote) [inline OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs]: I would prefer it if we could use an example that was entirely true.  Could you fix the after case to reflect what it actually does?

Comment (Phrohdoh) [inline OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs]: Sure, I'll have to move it above the `MiniYaml.MergeStrict`.

**Comment by Phrohdoh** [comment]
Updated the comment.

**Comment by abcdefg30** [comment]
Well, simple enough. :+1:

**Comment by obrakmann** [comment]
:+1:

**Pull request merged**